CI/travis/before_install_linux

```diff
@@ -31,7 +37,7 @@ handle_centos() {
 
 	# CENTOS 6 will has issues with sphinx
 	if is_centos_at_least_ver "7" ; then
-		yum -y install doxygen
+		yum -y install doxygen man2html
 		install_sphinx
 	fi
```

Comment:
there are 2 `if is_centos_at_least_ver "7" ; then` blocks
merge them?

Reply:
make sense. Will fix.

Reply:
those are actually in master, I will fix in a separate commit after this is on master.